src/app/book/book.js

```diff
@@ -6,9 +6,13 @@
     this.test = $location.path();
 
   }
+  function SuccessCtrl ($location) {
 
+    this.test = $location.path();
```

Comment:
get rid of this.test (and remove the $location as an argument.  this is not needed